.eslintrc.json

```diff
@@ -20,7 +20,8 @@
         "fixStyle": "inline-type-imports"
       }
     ],
-    "@typescript-eslint/no-unused-vars": "off",
+    "@typescript-eslint/no-unused-vars": "warn",
+    "@typescript-eslint/explicit-function-return-type": "off",
```

Comment:
Yea, it's a bit much, and I found it annoying at first, but it grew on me. It speeds up TSC and really forces us to define our API. Counter arguments?